Documentation/platforms/container-linux-systemd.md

```diff
@@ -0,0 +1,138 @@
+## Introduction
+
+The following guide shows how to run etcd with [systemd][systemd-docs] under [Container Linux][container-linux-docs].
+
+## etcd v0, v2
+
+[Container Linux][container-linux-docs] filesystem includes `etcd.service` for v0.x and `etcd2.service` for v2.x, both of which will be removed in later versions. See [Container Linux FAQ][container-linux-faq] for more detail.
```

Comment:
[Container Linux][container-linux-docs]'s include a unit file `etcd2.service` for etcd 2.x, which will be removed in the near future. See [Container Linux FAQ][container-linux-faq] for more details.